### DISCUSSION TITLE
Metformin and Cimetidine PBPK models

### DISCUSSION THREAD

**Discussion by NinaHanke** [Open-Systems-Pharmacology/Forum#732] [Announcement]
Dear all,

We are happy to share our carefully developed metformin and cimetidine PBPK models. The corresponding manuscript, including comprehensive documentation, has been accepted for publication in Clinical Pharmacokinetics and the models (Metformin, Cimetidine, Cimetidine-Metformin-DDI, Cimetidine-Midazolam-DDI) have been uploaded as OSP repositories (tags: osp-model, ddi, pk-sim, pbpk). 

Related publication: 
Hanke N, Türk D, Selzer D, Ishiguro N, Ebner T, Wiebe S, Müller F, Stopfer P, Nock V, Lehr T. 
A Comprehensive Whole‑Body Physiologically Based Pharmacokinetic Drug–Drug–Gene Interaction Model of Metformin and Cimetidine in Healthy Adults and Renally Impaired Individuals. 
Clinical Pharmacokinetics (2020), https://doi.org/10.1007/s40262-020-00896-w

We would be very pleased if some of you find these models useful.

Cheers, 
Nina

**Comment by StephanSchaller** [comment]
Congratulations!
